plugins/c8y_log_plugin/Cargo.toml

```diff
@@ -1,38 +1,42 @@
 [package]
-name = "tedge_logfile_request_plugin"
-version = "0.6.4"
+name = "c8y_log_plugin"
+version = "0.1.0"
```

Comment:
Every other plugin uses the same version as the core. I understand it doesn't strictly have to, but helps in avoiding version mismatches. So, any specific reason you changed that?

Reply:
No I frankly do not remember changing this, will revert.